### ISSUE TITLE
Adding RC release for 1.0

### ISSUE THREAD

**Issue by ToucheSir** [conda-forge/hydra-core-feedstock#1]
(Redirected from https://github.com/facebookresearch/hydra/issues/667)

Issue: RC releases on conda-forge

Really appreciate the publishing of stable releases on conda-forge! I was wondering if it would be possible to add an `rc` branch for the upcoming 1.0 release as well as per https://github.com/conda-forge/cfep/blob/master/cfep-05.md?

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
...
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       0_gnu    conda-forge
ca-certificates           2020.4.5.2           hecda079_0    conda-forge
certifi                   2020.4.5.2       py38h32f6830_0    conda-forge
hydra-core                0.11.3                     py_0    conda-forge
ld_impl_linux-64          2.34                 h53a641e_5    conda-forge
libffi                    3.2.1             he1b5a44_1007    conda-forge
libgcc-ng                 9.2.0                h24d8f2e_2    conda-forge
libgomp                   9.2.0                h24d8f2e_2    conda-forge
libstdcxx-ng              9.2.0                hdf63c60_2    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
omegaconf                 1.4.1                      py_2    conda-forge
openssl                   1.1.1g               h516909a_0    conda-forge
pip                       20.1.1                     py_1    conda-forge
python                    3.8.3           cpython_he5300dc_0    conda-forge
python_abi                3.8                      1_cp38    conda-forge
pyyaml                    5.3.1            py38h1e0a361_0    conda-forge
readline                  8.0                  hf8c457e_0    conda-forge
setuptools                47.1.1           py38h32f6830_0    conda-forge
six                       1.15.0             pyh9f0ad1d_0    conda-forge
sqlite                    3.30.1               hcee41ef_0    conda-forge
tk                        8.6.10               hed695b0_0    conda-forge
typing-extensions         3.7.4.2                       0    conda-forge
typing_extensions         3.7.4.2                    py_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
xz                        5.2.5                h516909a_0    conda-forge
yaml                      0.2.5                h516909a_0    conda-forge
zlib                      1.2.11            h516909a_1006    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : ...
    active env location : $HOME/anaconda3/envs/...
            shell level : 1
       user config file : $HOME/.condarc
 populated config files : $HOME/.condarc
          conda version : 4.8.3
    conda-build version : 3.18.8
         python version : 3.7.3.final.0
       virtual packages : __cuda=10.2
                          __glibc=2.23
       base environment : $HOME/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : $HOME/anaconda3/pkgs
                          /home/.../.conda/pkgs
       envs directories : $HOME/anaconda3/envs
                          $HOME/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.3 requests/2.23.0 CPython/3.7.3 Linux/4.15.0-99-generic ubuntu/16.04.5 glibc/2.23
                UID:GID : 1006:1006
             netrc file : None
           offline mode : False
```
</details>

**Comment by ocefpaf** [comment]
Sure. You'll need to create a branch and follow the instructions in that pep to add the `conda_build_config.yaml` pointing to the rc label.

**Comment by mdraw** [comment]
I've created an rc branch and a PR for this at #2. It depends on an unreleased rc version of OmegaConf though. I'd just wait for the release of stable OmegaConf 2.0.1 before merging the hydra 1.0.0rc1 PR or if someone needs it urgently, feel free to submit an update PR to the rc branch of https://github.com/conda-forge/omegaconf-feedstock (I currently don't have the time to test it properly and update myself).

**Comment by omry** [comment]
@mdraw, Releasing 2.0.1rc should be very smooth (it is nearly identical to 2.0.0).
I am holding off OmegaConf 2.0.1 because Hydra 1.0 is still maturing.

**Comment by omry** [comment]
@mdraw , now that Hydra 1.0 is out - can you update it here please?

**Comment by maclandrol** [comment]
@mdraw  ? If you are too busy maybe add @omry or someone else as maintainer ?

**Comment by mdraw** [comment]
Resolved by #7
